src/box/lua/net_box.lua

```diff
@@ -1351,6 +1504,19 @@ function remote_methods:_install_schema(schema_version, spaces, indices,
     self._on_schema_reload:run(self)
 end
 
+-- stream methods
+stream_methods.call = remote_methods.call
+stream_methods.call_16 = remote_methods.call_16
+stream_methods.close = remote_methods.close
+stream_methods.eval = remote_methods.eval
+stream_methods.eval_16 = remote_methods.eval_16
+stream_methods.execute = remote_methods.execute
+stream_methods.eval = remote_methods.eval
+stream_methods.ping = remote_methods.ping
+stream_methods.prepare = remote_methods.prepare
+stream_methods.unprepare = remote_methods.unprepare
+stream_methods._request = remote_methods._request
+
```

Comment:
Reusing remote_methods as is looks fragile. Should anyone add some new logic to e.g. _request that depends on new connection fields, stream might get broken.

Reply:
See my answer for previous comment, we need to discuss it